README.md

```diff
@@ -65,7 +65,7 @@ Follow the installation instructions below to initialize a development virtual e
 
 ### Development Environment Installation using `uv`
 
-GT4Py uses the [`uv`](https://docs.astral.sh/uv/) project manager for the development workflow. `uv` is a versatile tool that consolidates functionality usually distributed across different applications into subcommands.
+GT4Py uses the [`uv`](https://docs.astral.sh/uv/) project manager (`uv>=0.5.25`) for the development workflow. `uv` is a versatile tool that consolidates functionality usually distributed across different applications into subcommands.
```

Comment:
I don't like putting the version here, because we will randomly update. Can we instead write, please use a recent version of uv?